src/torch/1-split-encoder-classifier/split-encoder-classifier.lua

```diff
@@ -8,13 +8,14 @@
 -- to encode images.
 --
 
-local libsFolder = require('paths').thisfile('..')
-package.path = package.path .. ';' .. libsFolder .. '/0-tiefvision-commons/?.lua'
+local torchFolder = require('paths').thisfile('..')
+package.path = string.format("%s;%s/?.lua", os.getenv("LUA_PATH"), torchFolder)
```

Comment:
Torch sets up `LUA_PATH` (can be found in install/bin/torch-activate)
`package.path` is set to `LUA_PATH` when lua starts ([source](https://www.lua.org/manual/5.1/manual.html#pdf-package.path))